hub/templates/hub/area.html

```diff
@@ -151,6 +151,18 @@ <h5 class="card-title mb-3">Age range</h5>
                     </div>
                 </div>
             </div>
+            <div class="col d-flex flex-column">
+                <div class="card flex-grow-1 fake-data">
+                    <div class="card-body">
+                        <h5 class="card-title mb-3">Fuel poverty</h5>
```

Comment:
Maybe this title should be "Households in fuel poverty"? Otherwise a "Fuel poverty" figure of, say, "21%" is a bit ambiguous.